### DISCUSSION TITLE
Support for global `~/.gitignore`

### DISCUSSION THREAD

**Discussion by pepelsbey** [11ty/eleventy#4043] [Enhancement Queue]
### Is your feature request related to a problem? Please describe.

I get the passthrough copy running from time to time because I navigate/edit the src folder in Finder. Though it’s not a huge problem, I expect Eleventy to recognize global .gitignore, just like Git does.

Here’s my [global ~/.gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files?platform=mac#configuring-ignored-files-for-all-repositories-on-your-computer) file:

```
*.pyc
.DS_Store
Desktop.ini
._*
Thumbs.db
.Spotlight-V100
.Trashes
.idea
Icon
```

…properly linked in ~/.gitconfig:

```
[core]
	excludesfile = /Users/pepelsbey/.gitignore
```

But sometimes I see .DS_Store files in logs:

```
[11ty] Passthrough copy file added: src/articles/skewed-highlight/.DS_Store
```

### Describe the solution you'd like

It would be good if Eleventy supported global .gitignore rules, not just the local ones.

**Comment by zachleat** [comment]
Putting this one in the queue!

> This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

> View [the enhancement backlog here](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+). Don’t forget to upvote the top comment with 👍!
